classes/class-wc-connect-service-schemas-store.php

```diff
@@ -195,5 +195,16 @@ public function get_predefined_packages_schema() {
 
 			return ( object ) $predefined_packages;
 		}
+
+		public function get_predefined_packages_schema_for_service( $service_id ) {
+			$package_schemas = $this->get_predefined_packages_schema();
+
+			if ( ! isset( $package_schemas )
+			     || ! property_exists( $package_schemas, $service_id ) ) {
+				return array();
+			}
+
+			return $package_schemas->$service_id;
```

Comment:
Just a matter of style, but I'd prefer to see curly braces surrounding variable object properties. e.g. `$package_schemas->{$service_id}`.